app/src/main/java/com/dsu/pocapplication/news/ui/login/LoginFragment.kt

```diff
@@ -0,0 +1,4 @@
+package com.dsu.pocapplication.news.ui.login
+
+class LoginFragment {
+}
```

Comment:
Both activity file and fragment file shoule not be required. One among the two should do. Please decide and delete the respective file. If you are confused on activities and fragments, you can explore on the same. Here is a start: https://stackoverflow.com/questions/10478233/why-fragments-and-when-to-use-fragments-instead-of-activities

Reply:
done